modules/ossm-operatorhub-remove.adoc

```diff
@@ -0,0 +1,90 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_install/removing-ossm.adoc
+
+[id="ossm-operatorhub-remove_{context}"]
+= Removing the Operators from OperatorHub
```

Comment:
Maybe, `Removing the Operators Installed from OperatorHub`

Reply:
I changed this to read:

"Removing the installed Operators"